packages/sw-routing/README.md

```diff
@@ -8,159 +8,305 @@ A service worker helper library to route request URLs to handlers.
-to those Route in the order they are registered.
+[packages/sw-routing/src/lib/router.js:50-148](https://github.com/GoogleChrome/sw-helpers/blob/b84c1e5cba772aac25825aa6b88df832294d7113/packages/sw-routing/src/lib/router.js#L50-L148 "Source code on GitHub")
+
+The Router takes one or more [Routes][Route](#route) and registers a [`fetch`
+event listener](https://developer.mozilla.org/en-US/docs/Web/API/FetchEvent)
+that will respond to network requests if there's a matching route.
+
+It also allows you to define a "default" handler that applies to any requests
+that don't explicitly match a `Route`, and a "catch" handler that responds
+to any requests that throw an exception while being routed.
+
+**Examples**
+
+```javascript
+// The following example sets up two routes, one to match requests with
+// "assets" in their URL, and the other for requests with "images", along
+// different runtime caching handlers for each.
+// Both the routes are registered with the router, and any requests that
+// don't match either route will be handled using the default NetworkFirst
+// strategy.
+const assetRoute = new RegExpRoute({
+  regExp: /assets/,
+  handler: new goog.runtimeCaching.StaleWhileRevalidate(),
+});
+const imageRoute = new RegExpRoute({
+  regExp: /images/,
+  handler: new goog.runtimeCaching.CacheFirst(),
+});
+
+const router = new goog.routing.Router();
+router.registerRoutes({routes: [assetRoute, imageRoute]});
+router.setDefaultHandler({handler: new goog.runtimeCaching.NetworkFirst()});
+```
 
 ### constructor
 
-[packages/sw-routing/src/lib/express-route.js:36-43](https://github.com/GoogleChrome/sw-helpers/blob/a6f471056b52998c278dcb22dfc96169ea99932a/packages/sw-routing/src/lib/express-route.js#L36-L43 "Source code on GitHub")
+[packages/sw-routing/src/lib/express-route.js:65-94](https://github.com/GoogleChrome/sw-helpers/blob/b84c1e5cba772aac25825aa6b88df832294d7113/packages/sw-routing/src/lib/express-route.js#L65-L94 "Source code on GitHub")
 
 **Parameters**
 
 -   `path` **[string](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String)** The path to use for routing.
--   `handler` **[function](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/function)** The handler to manage the response.
+           If the path contains [named parameters](https://github.com/pillarjs/path-to-regexp#named-parameters),
+           then an Object mapping parameter names to the corresponding value
+           will be passed to the handler via `params`.
+-   `handler` **[Object](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object)** An Object with a `handle` method. That method
+           will be used to respond to matching requests.
+-   `method` **\[[string](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String)]** Only match requests that use this
+           HTTP method. Defaults to `'GET'` if not specified.
```

Comment:
The point of quotation marks is to set the thing off visually, believe it or not. We get that by virtue of the monospaced font. If it's not part of the literal, take it out.